app/configurator/map/map-chart-options.tsx

```diff
@@ -104,7 +104,8 @@ const BaseLayerSettings = memo(() => {
           field={null}
           path="baseLayer.show"
         />
-        <Flex>
+        {/* FIXME: change the styles of MUI components to match design specs */}
```

Comment:
Which MUI components ?

Reply:
In the map control design file on Sketch, for example the Radio Button looks differently and also the margin between button and its label weren't matching the specs. Also I had some problems to align the buttons in this case, as for example the radio button has its own internal padding of 9px, while checkbox does not. If they are used just as <Checkbox /><RadioButton /> next to each other, the aren't aligned correctly. I think it would make sense to align this and also check other chart options components, as I think there are also some inconsistencies. I think it would make sense to include this in potential technical improvements.

Reply:
OK thanks for the explanation, can you add those tasks there : https://github.com/visualize-admin/visualization-tool/issues/620 ?

Reply:
I will, yes!